main.go

```diff
@@ -1073,31 +1073,30 @@ func getBuildParameters(conf *config, build *buildOptions) ([]*gitlab.PipelineVa
 	return buildParameters, nil
 }
 
-// stopBuildsOfMergedPRs stops any running pipelines on a PR which has been merged.
-func stopBuildsOfMergedPRs(pr *github.PullRequestEvent, conf *config) error {
+// stopBuildsOfStalePRs stops any running pipelines on a PR which has been merged.
+func stopBuildsOfStalePRs(pr *github.PullRequestEvent, conf *config) error {
 
-	// If the action is "closed" and the "merged" key is "true", the pull request was merged.
-	// While webhooks are also triggered when a pull request is synchronized, Events API timelines
-	// don't include pull request events with the "synchronize" action.
-	if !(pr.GetAction() == "closed" && pr.PullRequest.GetMerged()) {
-		log.Debugf("stopBuildsOfMergedPRs: PR not merged and closed")
+	// If the action is "closed" the pull request was merged or just closed,
+	// stop builds in both cases.
+	if !(pr.GetAction() == "closed") {
+		log.Debugf("stopBuildsOfStalePRs: PR not merged and closed")
```

Comment:
Maybe not the correct log message anylonger, how about:
```suggestion
		log.Debugf("stopBuildsOfStalePRs: PR not closed, therefore not stopping it's pipeline")
```
Or something along these lines.